test/package-sources/microshift-latest-ec.toml

```diff
@@ -0,0 +1,7 @@
+id = "microshift-latest-ec"
+name = "Latest MicroShift Engineering Candidate"
+type = "yum-baseurl"
+url = "https://mirror.openshift.com/pub/openshift-v4/${UNAME_M}/microshift/ocp-dev-preview/latest-4.14/elrhel-9/os/"
```

Comment:
Just FYI, we _might_ need to change the `elrhel-9` portion of this path back to just `el9` later. @vfreex was going to look at fixing the pipeline change that introduced that path change for EC4.